asab/api/service.py

```diff
@@ -23,6 +25,19 @@ def __init__(self, app, service_name="asab.ApiService"):
 		self.ZkContainer = None
 		self.AttentionRequired = {}  # dict of errors found.
 
+		path = asab.Config.get("general", "manifest")
+		if len(path) == 0:
+			if os.path.isfile("/MANIFEST.json"):
+				path = "/MANIFEST.json"
+			elif os.path.isfile("MANIFEST.json"):
+				path = "MANIFEST.json"
+		try:
+			with open(path) as f:
```

Comment:
`if len(path) == 0: manifest = dict`

This code HAS TO run on systems, where there is no MANIFEST without error.

Reply:
@ateska I throw this warning if its missng

Loading MANIFEST.json failed for reason [Errno 2] No such file or directory: '/MANIFEST.json'

Do you think it is the correct action?

Reply:
Let's have a quick call on this.
We  need to maintain backward compatibility without any warnings.

Reply:
@ateska Sure when can we have the call?